src/components/notifications/index.tsx

```diff
@@ -28,7 +28,7 @@ export default (): JSX.Element => {
     const windowMonitor = Variable.derive(
         [bind(hyprlandService, 'focusedMonitor'), bind(monitor), bind(active_monitor)],
         (focusedMonitor, monitor, activeMonitor) => {
-            if (activeMonitor === true) {
+            if ((activeMonitor === true) && (focusedMonitor != undefined)) {
```

Comment:
```suggestion
            if ((activeMonitor === true) && (focusedMonitor)) {
```
Same as above but a truth check instead.